base_layer/core/src/transactions/transaction/mod.rs

```diff
@@ -183,6 +183,17 @@ impl OutputFeatures {
             ..Default::default()
         }
     }
+
+    pub fn for_checkpoint(merkle_root: Vec<u8>, asset_public_key: PublicKey) -> OutputFeatures {
+        Self {
+            flags: OutputFlags::ASSET_CHECKPOINT,
+            asset: Some(AssetOutputFeatures {
```

Comment:
This feature should only be used for AssetRegistration. The `parent_public_key` is the field that should be used. The `unique_id` field should also be populated with a specific value. Perhaps [0000....000] or [11111...111]?
```suggestion
            asset: None
```